### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by TeriyakiBomb** [kapalla/nova-laravel-pint#1]
When opening a project I get:

TypeError: undefined is not an object (evaluating 'nova.activeTextEditor.onDidSave')
 kapalla.pint/Scripts/main.js:73:18

So on this line: ```nova.workspace.activeTextEditor.onDidSave(format);```

**Comment by TeriyakiBomb** [comment]
I was playing around and this now works, not doing a pull request as this was a really lazy change and I haven't really spent much time looking through the extension (This code was lifted from the nova go formatter extension):

```
//nova.workspace.activeTextEditor.onDidSave(format);
  nova.workspace.onDidAddTextEditor((editor) => {
    if (editor.document.syntax === "php") {
      editor.onWillSave(format);
    }
  });```

**Comment by kapalla** [comment]
Thank you @TeriyakiBomb. Fixed it in v1.0.1
